contrib/devtools/previous_release.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2018-2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+#
+# Build previous releases.
+
+import argparse
+import contextlib
+from fnmatch import fnmatch
+import os
+from pathlib import Path
+import re
+import shutil
+import subprocess
+import sys
+import hashlib
+
+
+def usage(args, extras) -> int:
```

Comment:
I think python prints the help by itself already, no?

Reply:
removed usage in 9c34aff